django_tests/all_tests.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Running all_tests makes sense to me if that's how it's expected to work locally anyway. Just realized there's no .bat equivalent in the PR for Windows 😛